test/e2e/declarative_test_fwk/helper.py

```diff
@@ -110,21 +114,36 @@ def delete(self, reference: k8s.CustomResourceReference) -> None:
         sleep(self.DEFAULT_WAIT_SECS)
         self.wait_for_delete(reference)  # throws exception if wait fails
 
-    def assert_expectations(self, verb: str, input_data: dict, expectations: model.ExpectDict, reference: k8s.CustomResourceReference) -> None:
-        """Asserts custom resource reference inside Kubernetes cluster against the supplied expectations
+    def assert_k8s_resource(self, verb: str, input_data: dict, expectations_k8s: model.ExpectK8SDict, reference: k8s.CustomResourceReference) -> None:
+        """Compare the supplied expectations with custom resource reference inside Kubernetes cluster
 
-        :param verb: expectations after performing the verb (apply, patch, delete)
+        :param verb: compare expectations with k8s resource after performing the verb (apply, patch, delete)
```

Comment:
nit: We aren't using this verb parameter in this method

Reply:
Deleted them.